Sources/BraintreeCoreSwift/BTAPIClient.swift

```diff
@@ -22,6 +26,14 @@ import Foundation
     var apiHTTP: BTAPIHTTP?
     var graphQLHTTP: BTGraphQLHTTP?
 
+    /// Exposed for testing analytics
+    /// By default, the `BTAnalyticsService` instance is static/shared so that only one queue of events exists.
+    /// The "singleton" is managed here because the analytics service depends on `BTAPIClient`.
+    var analyticsService: BTAnalyticsService? {
```

Comment:
Good call - yeah it was present in Obj-C as well. My initial thought would be to make this one weak but I feel like from an ownership perspective it could potentially go either way. Thoughts?

Reply:
I agree - a weak reference to `BTAnalyticsService` makes sense to me.